boilr.go

```diff
@@ -1,22 +1,22 @@
 package main
 
 import (
-  "fmt"
-  "github.com/tmrts/boilr/pkg/boilr"
-  "github.com/tmrts/boilr/pkg/cmd"
-  "github.com/tmrts/boilr/pkg/util/exit"
-  "github.com/tmrts/boilr/pkg/util/osutil"
+	"fmt"
+
+	"boilr/pkg/boilr"
+	"boilr/pkg/cmd"
+	"boilr/pkg/util/exit"
+	"boilr/pkg/util/osutil"
 )
 
 func main() {
-  if exists, err := osutil.DirExists(boilr.Configuration.TemplateDirPath); ! exists {
-    if err := osutil.CreateDirs(boilr.Configuration.TemplateDirPath); err != nil {
-      exit.Error(fmt.Errorf("Tried to initialise your template directory, but it has failed: %s", err))
-    }
-  } else if err != nil {
-    exit.Error(fmt.Errorf("Failed to init: %s", err))
-  }
+	if exists, err := osutil.DirExists(boilr.Configuration.TemplateDirPath); !exists {
```

Comment:
Hmm whats going on here it looks like we didn't change anything of significance and we reformatted the file. If this isn't important I say we dont submit this change.

Reply:
this was automatically reformated by Intellij idea, I checked with other files in this project and those are using `tabs` indentation, this file was using 2 spaces for it, so I think is good to keep these changes as they adjust that to use the project standard.